src/content/docs/style-guide/images/embed-images.mdx

```diff
@@ -78,50 +131,44 @@ Descriptive captions help the reader know why the image matters. If it's a scree
 
 For more help with captions and other supporting text around images, see [Guidelines for explaining images](/docs/style-guide/writing-guidelines/screenshots-images#identify-images).
 
-## Add an inline image [#inline-images]
+## Add an inline image (MDX) [#inline-images-mdx]
 
-To use an inline image, add something like this:
+To use an inline image:
+* import the image at the top of the file
+* use the `img` tag in the file
+* add a class of `inline` to `img`
+* add any optional height/width in the `style` prop
```

Comment:
Not sue what "prop" means. Property?

Reply:
props (in JSX) or attributes (in HTML) would things like `class`, `style`, `alt`, `title` in the tag like so:

```jsx
<img alt="text" title="text" style={{height: '29px'}} class="inline" />
```